sirepo/package_data/static/js/sirepo-utils.js

```diff
@@ -11,6 +11,32 @@ class SirepoUtils {
         return s.charAt(0).toUpperCase() + s.slice(1);
     }
 
+    static orderOfMagnitude(val, binary=false) {
+        const MAGS = ['', 'k', 'M', 'G', 'T', 'P', 'E'];
```

Comment:
Since this supports binary, it should use Ki, Mi, Gi, Ti, Pi, Ei.

Reply:
The "i" is added on returning, but also [d8a178b](https://github.com/radiasoft/sirepo/pull/4736/commits/d8a178b58ca0e460e6e7c512b169e0dba74f0c53)